src/main/java/com/minecolonies/util/Utils.java

```diff
@@ -365,6 +368,7 @@ public static boolean isTool(@Nullable ItemStack itemStack, String toolType)
      * @param tool  the tool category
      * @return integer value for mining level &gt;= 0 is okay
      */
+    @SuppressWarnings("deprecation")
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Suppressing warnings is not allowed [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1309)